blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/BluefloodServiceStarter.java

```diff
@@ -313,4 +316,20 @@ public void run() {
         startEventListenerModules();
         log.info("All blueflood services started");
     }
+
+    @VisibleForTesting
+    public static boolean SkipInstantiateRestartGauge = false;
+
+    @VisibleForTesting
+    public static boolean ThrowInsteadOfExit = false;
```

Comment:
Excellent. I didn't find anything in the oracle docs about trapping `System.exit`, but these other things will do. I'll use them. I'm not a big fan of this either.